### ISSUE TITLE
cuDNN failed to initialize

### ISSUE THREAD

**Issue by xiaoleiwang10** [CSBDeep/CSBDeep_website#12]
Dear everyone,
When I test the example about probilistic neural networking and run "restored = model.predict(x, axes, normalizer=None)", I got error. The error said: (0) Unknown: Failed to get convolution algorithm. This is probably because cuDNN failed to initialize, so try looking to see if a warning log message was printed above. [[{{node down_level_0_no_0_1/convolution}}]].
My software versions are:
Cuda10.1
cudnn7.6.3.30
Tensorflow-gpu:the latest version.

Do you know what's the problem? Any information would be helpful!

Thank you,
Xiaolei
